### DISCUSSION TITLE
Basic authentication logic not understanding

### DISCUSSION THREAD

**Discussion by azeng528** [Kong/kong#13415] [Help]
1、There are two routers and each router is configured with an auth plugin
![image](https://github.com/user-attachments/assets/42d2675c-885b-4674-9d9a-1189a0dca96c)
2、have two consumer
![image](https://github.com/user-attachments/assets/34ade88c-0335-433b-b7c7-6a8a17e3ec0d)
3、test
Using the authentication information of any one consumer can access the interfaces corresponding to two routers, but it is not possible to fix the binding relationship between the consumer and the router/plugin

**Comment by xianghai2** [comment]
Could you provide detailed steps to reproduce this issue? thanks.
